### ISSUE TITLE
Adding User's groups as drop-down - feature request

### ISSUE THREAD

**Issue by romainGuiet** [tlambert03/napari-omero#31]
Dear @tlambert03 ,

I found a bit "confusing" to have a "flat" access to all the projects when connecting to the session.
As the default omero behavior is to arrive in your "default" group, and to be able to move/jump to another group, I would suggest to have a drop-down menu to list all the groups individually (and maybe keep a All Groups option)

Something like :
- Default group
- Another group
- Yet another group
- Display all groups

Cheers,

Romain

**Comment by jo-mueller** [comment]
@tlambert03 As this is a quite old issue I was wondering whether this is planned in the near future? If youpoint me to the right place, I can also try myself at a PR to implement this.

**Comment by Rdornier** [comment]
Hello, 

I agree with Romain suggestion. It would be great to select the group we want to browse (and maye be also the user because with Read-Only or higer permission groups, we also have access to teammate's data). I found interesting the way that [QuPath-OMERO extension](https://github.com/qupath/qupath-extension-omero) implements [OMERO data browsing](https://github.com/qupath/qupath-extension-omero/blob/main/src/main/java/qupath/lib/images/servers/omero/OmeroWebImageServerBrowserCommand.java#L227). A similar implementation would be great.

Here is a screenshot of what the QuPath-OMERO brower looks like.

![image](https://user-images.githubusercontent.com/64911638/228213101-2c852307-513f-4f85-b3f2-b9eefbe89656.png)

Hope it helps,
Best,

Rémy.

**Comment by tlambert03** [comment]
thanks all, I've got a branch roughly working... will continue work on it soon

**Comment by romainGuiet** [comment]
Hi @tlambert03 ,

thank you so much for taking time for working on this ! 

I just tested the "projects" branch and when logginh for the first time I see all the projects from all members of my default group (see below)
<img width="885" alt="image" src="https://user-images.githubusercontent.com/8309560/231659599-6673fdc1-dcf1-442f-b84a-7770d5105d92.png">

but if I switch the "User" param or the "Group" param it's "updated" and now it shows only my projects 
<img width="885" alt="image" src="https://user-images.githubusercontent.com/8309560/231659846-39bcf201-4cbc-4723-9b39-92ae350ea0de.png">

🥳

**Comment by tlambert03** [comment]
Yep, it's close ... some bugs I still need to iron out before I'm comfortable merging it

**Comment by jo-mueller** [comment]
@tlambert03 do you ba any chance rememer what it was that was holding this from going through? It looks quite functional as far as I can see.

**Comment by tlambert03** [comment]
I don't remember, I'm sorry.  But you can feel free to merge it if it does what you want!

**Comment by jo-mueller** [comment]
Will do! I thinkI spotte some minor bugs, but I think it would be great to have this functionality for the time being and then take it from there 👍

**Comment by psobolewskiPhD** [comment]
I should have some time in the next couple weeks to test or look at this too.
Would be super valuable!

**Comment by jo-mueller** [comment]
Cool, I'll merge then. Thanks in advance for the extra pair of eyes!
